docker/test_dockerimage.sh

```diff
@@ -13,5 +13,15 @@ emcc --version
 java -version
 cmake --version
 
+mkdir -p test
+
+# test emcc compilation
+echo 'int main() { return 0; }' > test/main.c
+emcc -c test/main.c -o test/main
+
+# test embuilder
+embuilder build zlib
+
 # cleanup after test
+rm -rf test
 find ${EMSDK} -name "*.pyc" -exec rm {} \;
```

Comment:
Why cleanup here at all..  isn't this run with `--rm` meaning its just gets thrown away anyway?

Reply:
The `--rm` parameter will not remove files within the mounted volume. I removed these cleanups with commit https://github.com/emscripten-core/emsdk/pull/571/commits/d7021674357cf0d3bfa352e7eafc2733c2de32a7 and compiled the test to `/tmp` instead.